docs/guides/developer/docs/api/types.md

```diff
@@ -0,0 +1,339 @@
+catalogs:
+
+ - The default metadata catalogs for events (`dublincore/episode`) and series (`dublincore/series`)
+ - An arbitrary number of configurable extended metadata catalogs can be configured for both events and series
+
+While the extended metadata can be fully configured, the default metadata catalogs are supposed to hold a minimum
+set of defined metadata fields.
+
+As the metadata catalogs are configurable, the Application API provides means of gathering the metadata catalog
+configuration.
+
+This is done by ["/api/events/{event_id}/metadata"](events-api.md#metadata) and
+["/api/series/{series_id}/metdata"](series-api.md#metadata). Those requests return self-describing metadata catalogs
+that do not just contain the values of all metadata fields but also a list of available metadata fields and their
+configuration.
+
+Note that the Opencast configuration defines which metadata catalogs are available for events and series.
+
+The following sections define data types that are used to manage metadata catalogs.
+
+### fields
+
+Each metadata catalogs has a list of metadata fields that is described as JSON objects with the following fields:
+
+Field          | Optional | Type                | Description
+:--------------|:---------|:--------------------|:-----------
+`id`           | no       | [`string`](#basic)  | Technical identifier of the metadata field
+`value`        | no       | [`string`](#basic)  | Current value of the metadata field
+`label`        | no       | [`string`](#basic)  | Displayable name of the metadata field (i18n)
+`type`         | no       | [`string`](#basic)  | Type of the metadata field
+`readOnly`     | no       | [`boolean`](#basic) | Whether this metadata field is read-only
+`required`     | no       | [`boolean`](#basic) | Whether this metadata field is mandatory
+`collection`   | yes      | [`string`](#basic)  | Pre-defined list of values as JSON object
+`translatable` | yes      | [`boolean`](#basic) | Whether the field `value` supports i18n
+
+__Example__
+
+```
+[
+  {
+    "readOnly": false,
+    "id": "title",
+    "label": "EVENTS.EVENTS.DETAILS.METADATA.TITLE",
+    "type": "text",
+    "value": "Test Event Title",
+    "required": true
+  },
+  {
+    "translatable": true,
+    "readOnly": false,
+    "id": "language",
+    "label": "EVENTS.EVENTS.DETAILS.METADATA.LANGUAGE",
+    "type": "text",
+    "value": "",
+    "required": false
+  },
+  {
+    "translatable": true,
+    "readOnly": false,
+    "id": "license",
+    "label": "EVENTS.EVENTS.DETAILS.METADATA.LICENSE",
+    "type": "text",
+    "value": "",
+    "required": false
+  },
+  [...]
+]
+```
+
+### values
+
+To modifiy values of metadata catalogs, a JSON array with JSON objects contained the following fields is used:
+
+Field   | Required | Description
+:-------|:---------|:-----------
+`id`    | yes      | The `id` of the metadata field
```

Comment:
Either the second `id` should not be in code, or the next line's value should be `value`

Reply:
Good catch, I've changed the second.